torch/_inductor/codegen/wrapper.py

```diff
@@ -76,6 +79,16 @@ def convert_arg_type(python_type):
         cpp_optional_type = PYTHON_TO_CPP[optional_type]
         return f"c10::optional<{cpp_optional_type}>"
 
+    # Convert arg of type List[*]
```

Comment:
It seems that the logic to handle `Optional[*]` and `List[*]` are similar.
Is it possible to add a function that can be reused for both `Optional[*]` and `List[*]` to reduce the duplicated code here?